src/components/Avatar/_variables.scss

```diff
@@ -1,6 +1,7 @@
 @use 'sass:map';
 
 $sizes: (
+    '3xs': 16px,
     '2xs': 20px,
     'xs': 24px,
     's': 28px,
```

Comment:
I thought that we wanted to change this sizes like in Button?

Reply:
Decided to make in the next major release